src/npg_porch/server.py

```diff
@@ -46,24 +51,37 @@
 app.include_router(pipelines.router)
 app.include_router(tasks.router)
 
+env = Environment(loader=PackageLoader("npg_porch", "templates"))
+templates = Jinja2Templates(env=env)
+
+version = metadata.version("npg_porch")
+
 
 @app.get(
     "/",
     response_class=HTMLResponse,
     tags=["index"],
+    summary="Web page with listing of Porch tasks.",
+)
+async def root(request: Request, db_accessor=Depends(get_DbAccessor)) -> HTMLResponse:
+    task_list: list[TaskExpanded] = await db_accessor.get_expanded_tasks()
+    return templates.TemplateResponse(
+        "index.j2",
+        {
+            "request": request,
```

Comment:
Do we need to pass request here and in teh code for the next endpoint?

Reply:
Yes, its a standard jinja requirement.